dev-libs/protobuf/protobuf-3.5.1.1.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="An efficient method of encoding structured data"
+DESCRIPTION="Protocol buffers are Google's language-neutral, platform-neutral, \
+extensible mechanism for serializing structured data – think XML, but smaller, \
+faster, and simpler. You define how you want your data to be structured once, \
+then you can use special generated source code to easily write and read your \
+structured data to and from a variety of data streams and using a variety of \
+languages – Java, C++, or Python."
+HOMEPAGE="https://github.com/google/protobuf"
+COPYRIGHT="2008-2018 Google"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="56b5d9e1ab2bf4f5736c4cfba9f4981fbc6976246721e7ded5602fbaee6d6869"
+PATCHES="protobuf-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion=15.0.1
+libVersionCompat="$libVersion compat >= 1"
+
+PROVIDES="
+	protobuf$secondaryArchSuffix = $portVersion
+	cmd:protoc$secondaryArchSuffix
+	lib:libprotobuf_lite$secondaryArchSuffix = $libVersionCompat
+	lib:libprotobuf$secondaryArchSuffix = $libVersionCompat
+	lib:libprotoc$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	protobuf${secondaryArchSuffix}_devel = $portVersion
+	devel:libprotobuf_lite$secondaryArchSuffix = $libVersionCompat
+	devel:libprotobuf$secondaryArchSuffix = $libVersionCompat
+	devel:libprotoc$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	protobuf$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	"
+
+defineDebugInfoPackage protobuf$secondaryArchSuffix \
+	$libDir/libprotobuf.so.$libVersion \
+	$libDir/libprotoc.so.$libVersion \
+	$libDir/libprotobuf-lite.so.$libVersion
+
+BUILD()
+{
+	libtoolize --force --copy --install
```

Comment:
`autoreconf`, which is called by `./autogen.sh`, will invoke `libtoolize` automatically.

```diff
-	libtoolize --force --copy --install